lasso-marko-plugin.js

```diff
@@ -95,18 +95,23 @@ module.exports = function(lasso, config) {
                 });
             } else {
                 if (meta.deps) {
-                    dependencies = dependencies.concat(meta.deps.map(depPath => ({
-                        type: 'require',
-                        path: require.resolve(this.resolvePath(depPath, nodePath.dirname(this.path)))
-                    })));
+                    dependencies = dependencies.concat(meta.deps.map(dep => (
+                        dep.code ? {
+                            type: dep.type,
+                            code: dep.code 
+                        } : {
+                            type: dep.includes(':') ? dep.slice(0, dep.indexOf(':')) : 'require',
```

Comment:
This is a micro-optimization, but it would be better to use `dep.indexOf(':')` instead of `dep.includes(':')` because you end up doing `dep.indexOf(':')` again later in the statement. Essentially looping through the `dep` twice.